lib/critcl/critcl.tcl

```diff
@@ -4936,7 +4936,9 @@ proc ::critcl::Cat {path} {
     # Easier to write our own copy than requiring fileutil and then
     # using fileutil::cat.
 
-    set fd [open $path r]
+    if {[catch {set fd [open $path r]} err]} {
+	return
+    }
```

Comment:
Can you provide examples of use where this is important ?
(Note that `Cat` is an internal command)
Also, should it really silently fail instead of providing an error about the missing file ?
(OK, changing from __missing file => fail__ to __missing file <=> empty file__ ?)

I would be more amenable to making the `Get` and `Put` commands in `lib/critcl-util/util.tcl` public, and having them in strict and non-strict variants (with the non-strict having the proposed semantics of `no file <=> empty file`.